addons/mail/models/ir_attachment.py

```diff
@@ -89,3 +93,15 @@ def _to_store_defaults(self):
             "type",
             "url",
         ]
+
+    def _attachment_ownership_access_token(self):
+        """Returns a scoped limited access token that indicates ownership of the attachment.
+        If verified by verify_limited_field_access_token, accessing the attachment bypasses
+        the ACLs.
+
+        :rtype: str
+        """
+        self.ensure_one()
+        return limited_field_access_token(
+            self, field_name="attachment_ownership", scope="attachment_ownership"
```

Comment:
For the field name in the other comment I had in mind the as_author_access_token -> ownership_token
Not the field name of the method here which can stay id I think.